### PR TITLE
chore: update kubernetes documentation

### PR DESCRIPTION
**Pull request by tpetillot**
**Modifications:**
* Remove node port
* Specify tolerations
* Specify local cluster mode behavior

### CODE REVIEW

File: content/reference/current/pools/kubernetes/index.md [hunk 1]
```diff
@@ -39,20 +54,35 @@ To configure the type of instances you want to spawn, you need to fill the form
...
 Limits and requests for memory are measured in bytes. You can express memory as a plain integer or as a fixed-point integer using one of these suffixes: E, P, T, G, M, K. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki.
 
-NOTE: If your Gatling Enterprise instance belongs to a Kubernetes cluster, you don't have to provide a **Kubernetes Url** and a **Service Account Token**.
-You can still choose to configure it, for example to create a pool in another cluster, by unticking `Use local cluster`.
-Also, you can specify preferring **Internal IP** over the **External IP** for connecting to Kubernetes nodes, by ticking `Prefer internal IP`.
+## Tolerations settings
+
+To configure [torelations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/), you may fill:
+
+{{< img src="tolerations-settings.png" alt="Kubernetes credentials" >}}
```
Comment (Isammoc) [inline content/reference/current/pools/kubernetes/index.md:65]: New image but no uploaded image?

File: content/reference/current/pools/kubernetes/index.md [hunk 2]
```diff
@@ -39,20 +54,35 @@ To configure the type of instances you want to spawn, you need to fill the form
...
 
-{{< img src="kubernetes-local.png" alt="Kubernetes local" >}}
```
Comment (Isammoc) [inline content/reference/current/pools/kubernetes/index.md:49]: Image removed in documentaiton, but image file changed?

**Comment by tpetillot** [comment]
Wooops! Some file didn't make it to the pull request!

File: content/reference/current/pools/kubernetes/index.md [hunk 3]
```diff
@@ -13,24 +13,39 @@ A Kubernetes/OpenShift Pool is a reference to your Kubernetes infrastructure.
...
-    - **TLS secret name**: the optional secret containing a certificate used by the ingress [TLS secrets documentation](https://kubernetes.github.io/ingress-nginx/user-guide/tls/#tls-secrets)
+      - **TLS secret name**: the optional secret containing a certificate used by the ingress [TLS secrets documentation](https://kubernetes.github.io/ingress-nginx/user-guide/tls/#tls-secrets)
     - **Route**: (OpenShift extension) exposes HTTP routes (HTTPS not supported) from outside the cluster to injectors within the cluster.
         - **Secured**: allow you to add the desired certificate on the route [OpenShift secured routes documentation](https://docs.openshift.com/container-platform/4.5/networking/routes/secured-routes.html)
         - **Certificate**: Certificate associated with the route
         - **Certificate key**: Certificate key associated to certificate
         - **CA Certificate**: Certificate authority signing the certificate
 
+{{< alert info >}}
+In **local cluster mode**, pods are access directly inside the cluster using service name and namespace.
```
Comment (slandelle) [inline content/reference/current/pools/kubernetes/index.md:44]: accessed

Comment (slandelle) [reply content/reference/current/pools/kubernetes/index.md:44]: the associated service name and namespace

File: content/reference/current/pools/kubernetes/index.md [hunk 4]
```diff
@@ -13,24 +13,39 @@ A Kubernetes/OpenShift Pool is a reference to your Kubernetes infrastructure.
...
+So **no need to configure outbound access** to the injectors through an ingress or a route, **field is hidden**.
```
Comment (slandelle) [inline content/reference/current/pools/kubernetes/index.md:45]: field is hidden => this part of the form is hidden

File: content/reference/current/pools/kubernetes/index.md [hunk 5]
```diff
@@ -39,20 +54,35 @@ To configure the type of instances you want to spawn, you need to fill the form
...
 
-{{< img src="kubernetes-local.png" alt="Kubernetes local" >}}
+To configure [torelations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/), you may fill:
+
+{{< img src="tolerations-settings.png" alt="Kubernetes credentials" >}}
+
+- **Key**: toleration key
+- **Operator**: Either exists or equal
+  - **Exists**: Any node matching the key will have the effect applied, no need to specify the value
+  - **Equal**: Any node matching the key with the given value will have the effect applied
```
Comment (slandelle) [inline content/reference/current/pools/kubernetes/index.md:70]: Are we sure those are the only possible values? Where does this come from?

**Review by slandelle — CHANGES_REQUESTED** [review]

File: content/reference/current/pools/kubernetes/index.md (hunk 5 above)
Comment (tpetillot) [reply content/reference/current/pools/kubernetes/index.md:70]: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/

> the operator is Exists (in which case no value should be specified), or
> the operator is Equal and the values are equal.

Comment (tpetillot) [reply content/reference/current/pools/kubernetes/index.md:70]: It's the same for other fields (e.g: effect which is specified aswell), and the link between the fields (we should have an ADT, where value is required only when Equal is selected as an operator).

Comment (slandelle) [reply content/reference/current/pools/kubernetes/index.md:70]: Do we want to change this right now? Or modify later?

**Review by slandelle — APPROVED** [review]

File: content/reference/current/pools/kubernetes/index.md (hunk 5 above)
Comment (tpetillot) [reply content/reference/current/pools/kubernetes/index.md:70]: Modify later 👍

**Pull request merged**